include/openrave/planner.h

```diff
@@ -507,14 +509,17 @@ class OPENRAVE_API PlannerStatus
         return statusCode&PS_HasSolution;
     }
 
-    PlannerParametersConstPtr parameters; ///< parameters used in the planner
-    std::string description;        ///< Optional, the description of how/why the error happended. Displayed to the user by the UI. It will automatically be filled with a generic message corresponding to statusCode if not provided.
-    uint32_t statusCode; // combination of PS_X fields (PlannerStatusCode)
-    IkParameterization ikparam;      // Optional,  the ik parameter that failed to find a solution.
-    std::vector<dReal> jointValues; // Optional,  the robot's joint values in rad or m
-    CollisionReportPtr report;       ///< Optional,  collision report at the time of the error. Ideally should contents contacts information.
-
-    std::string errorOrigin;        // Auto, a string representing the code path of the error. Automatically filled on construction.
+    PlannerParametersConstPtr parameters;   ///< parameters used in the planner
+    std::string description;                ///< Optional, the description of how/why the error happended. Displayed to the user by the UI. It will automatically be filled with a generic message corresponding to statusCode if not provided.
+    uint32_t statusCode;                    // combination of PS_X fields (PlannerStatusCode)
+    IkParameterization ikparam;             // Optional, the ik parameter that failed to find a solution.
+    std::vector<dReal> jointValues;         // Optional, the robot's joint values in rad or m
+    CollisionReportPtr report;              ///< Optional,  collision report at the time of the error. Ideally should contents contacts information.
+    // IMPORTANT: indices of vRobotJointValues and vCollidingBodies should correspond with each other!
+    std::vector< std::vector<dReal> > vRobotJointValues;                                 // Records robot joint values at all collisions
+    std::vector< std::vector< std::pair<std::string, std::string> > > vCollidingBodies;  // Records all collision reports
+
+    std::string errorOrigin;                // Auto, a string representing the code path of the error. Automatically filled on construction.
```

Comment:
Try to avoid these kind of cosmetic space changes, they make code reviewing much less efficient. It's hard to tell which ones you really changed and whether something is hidden in here.